bindings/python/microarch.py

```diff
@@ -85,6 +85,7 @@ def resolve_compiler(name: str):
     aliases = {
         "GNU": "gcc",
         "Clang": "clang",
+        "IntelLLVM": "intel"
```

Comment:
I think `intel` is the Intel® C++ Compiler Classic and that the name should be `oneapi` unless I misunderstood your setup.
https://github.com/archspec/archspec-json/pull/47
```suggestion
        "IntelLLVM": "oneapi"
```

Reply:
Yes you are right.